src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb

```diff
@@ -1216,12 +1220,8 @@ End Class
             comp.VerifyDiagnostics()
             comp.VerifyAnalyzerDiagnostics({New ParamsArrayTestAnalyzer}, Nothing, Nothing, False,
                                            Diagnostic(ParamsArrayTestAnalyzer.LongParamsDescriptor.Id, "M0(1, 2, 3, 4, 5)").WithLocation(9, 9),
```

Comment:
From what I saw in the analyzer code, `LongParamsDescriptor` is reported when an argument which is an array creation expression contains more than 3 elements in place for a param array parameter.

The analyzer used to check in both source order and in parameter order before, I  changed the analyzer to check only in evaluation order now, therefore those duplicate ones are removed.